apps/hosts/utils.py

```diff
@@ -1,6 +1,13 @@
 from .models import ChallengeHost
+from challenges.models import Challenge
 
 
 def get_challenge_host_teams_for_user(user):
     """Returns challenge host team ids for a particular user"""
     return ChallengeHost.objects.filter(user=user).values_list('team_name', flat=True)
+
+
+def is_user_a_host_of_challenge(user, challenge_pk):
+    """Returns boolean if the Participant Team participated in particular Challenge"""
```

Comment:
I think it will be more appropriate if we use `Participant` instead of `Participant Team`. ?